tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,189 @@
+"""CLI tests for logging.
+
+:Requirement: FIXME
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+import logging
+from robottelo import ssh
+from robottelo.test import CLITestCase
+from robottelo.cli.factory import (
+    make_org,
+    make_product_wait,
+    make_repository,
+)
+from fauxfactory import gen_string
+log = logging.getLogger(__name__)
+
+
+class SimpleLoggingTestCase(CLITestCase):
+    """Test class for default logging to files."""
+
+    org = None
+    product = None
+
+    def setUp(self):
+        """Tests for logging to files"""
+        super(SimpleLoggingTestCase, self).setUp()
+
+        # need own org for the manifest refresh test
+
+        if SimpleLoggingTestCase.org is None:
+            SimpleLoggingTestCase.org = make_org(cached=True)
+        if SimpleLoggingTestCase.product is None:
+            SimpleLoggingTestCase.product = make_product_wait(
+                {u'organization-id': SimpleLoggingTestCase.org['id']},
+            )
+
+    def _make_repository(self, options=None):
+        """Makes a new repository and asserts its success"""
+        if options is None:
+            options = {}
+
+        if options.get('product-id') is None:
+            options[u'product-id'] = self.product['id']
+
+        return make_repository(options)
+
+    def test_positive_logging_from_foreman_core(self):
+        """Check that GET command to Hosts API is logged and has request ID."""
+        GET_line_found = 0
+        # log the start of the test in case of problems with slow systems and time zones
+        self.logger.info("Testing test_positive_logging_from_foreman_core")
+        with ssh.get_connection() as connection:
+            result = connection.run('hammer host list')
+            self.assertEqual(result.return_code, 0)
+            # extract last ten lines from log
+            result = connection.run('tail -20 /var/log/foreman/production.log > \
+                        /var/tmp/logfile_from_foreman_core')
+            self.assertEqual(result.return_code, 0)
+        # use same location on remote and local for log file extract
+        logfile_location = '/var/tmp/logfile_from_foreman_core'
```

Comment:
Move that variable above and use it in `connection.run()`:
```python
            result = connection.run(('tail -20 /var/log/foreman/production.log > '
                        '{}'.format(logfile_location)))
```

Reply:
I can use it higher up too.